package.json

```diff
@@ -1,7 +1,4 @@
 {
-  "name": "gitea",
```

Comment:
Why remove these lines?

Reply:
They are not used anywhere and just noise.

BTW, there is no chance of accitental publish either: https://www.npmjs.com/package/gitea.